packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -23,16 +25,25 @@ export interface IComponentWrapperProps<T> {
     singleRow?: boolean;
     designSystem: T;
     view: ComponentViewTypes;
+    transparent: boolean;
```

Comment:
I'm not sure this should be a required prop - Seems more like something you would opt into.